README.md

```diff
@@ -331,26 +333,21 @@ For examples of using the `hexagon` scheduling directive on both the simulator a
 Hexagon DSP, see the blur example app.
 
 To build and run an example app using the Hexagon target,
-  1. Obtain and build LLVM and Clang v5.0 or later from llvm.org
+  1. Obtain and build LLVM and Clang v8.0 or later from llvm.org
   2. Download and install the Hexagon SDK and version 8.0 Hexagon Tools
   3. Build and run an example for Hexagon HVX
 
-#### 1. Obtain and build LLVM and clang v5.0 or later from llvm.org
-The Hexagon backend is currently under development. So it's best to use trunk llvm.
-These are the same instructions as above for building Clang/LLVM, but for trunk
-Clang/LLVM instead of 5.0.
-
-    cd <path to llvm>
-    svn co http://llvm.org/svn/llvm-project/llvm/trunk .
-    svn co http://llvm.org/svn/llvm-project/cfe/trunk ./tools/clang
-    # Or:
-    #    git clone http://llvm.org/git/llvm.git .
-    #    git clone http://llvm.org/git/clang.git llvm/tools
-    mkdir build
-    cd build
-    cmake -DLLVM_ENABLE_TERMINFO=OFF -DLLVM_TARGETS_TO_BUILD="X86;ARM;NVPTX;AArch64;Mips;PowerPC;Hexagon" -DLLVM_ENABLE_ASSERTIONS=ON -DCMAKE_BUILD_TYPE=Release ..
-    make -j8
-    export LLVM_CONFIG=<path to llvm>/build/bin/llvm-config
+#### 1. Obtain and build LLVM and clang v8.0 or later from llvm.org
+The Hexagon backend is currently under development, so it's best to use trunk llvm. (These are the same instructions as above for building Clang/LLVM.
```

Comment:
I really think we should make this more instructive rather than a suggestion to use trunk llvm. The reason is that for the longest time we have had a buildbot that has tested Hexagon only with trunk llvm and nothing else. So, I have no idea what'll happen if we even try to stick to the policy of supporting the last two releases of LLVM.

Reply:
I'm confused; what is the recommendation we are trying to make here?

Reply:
"If you wish to use Halide to target Hexagon with HVX, you should use trunk LLVM"

Reply:
PTAL